coramin/third_party/tests/test_minlplib_tools.py

```diff
@@ -0,0 +1,143 @@
+import coramin
+import unittest
+import os
+
+
+class TestMINLPLibTools(unittest.TestCase):
+    def test_get_minlplib_instancedata(self):
+        current_dir = os.path.abspath(os.path.dirname(__file__))
```

Comment:
I generally prefer `pyomo.common.fileutils.this_file_dir()`, as it is a little more robust to things like Windows and chdir.

Reply:
Fixed.